tests/test_backend.py

```diff
@@ -3304,7 +3304,7 @@ def func(input_x, block_size, pad):
 
     @check_opset_min_version(11, "BatchToSpaceND")
     def test_batch_to_spacend_non_const_7d(self):
-        x_type, y_type, z_type = np.int64, np.int64, np.int64
+        x_type, y_type, z_type = np.float32, np.int64, np.int64
```

Comment:
I believe we need `block_shape[0] == block_shape[1]` and `len(block_shape) == 2` to ensure the code you added to BatchToSpaceND is tested.  The block shapes here are only `[2, 3]`, and `[2]`.  Are you sure your code is being tested?

Reply:
good catch. Thanks.